### PR TITLE
Lab 3 assignment

### PR DESCRIPTION
**Pull request by Bear-03**

### CODE REVIEW

**Review by bardhprenkaj — APPROVED** [review]
No hyperparameter optimization has been done here, so that'll reflect your final score. I still take the solution as valid.

**Comment by Bear-03** [comment]
What about the "Optimization" section? There I optimize the model trying different values for dropout and learning rates, optimizer, number of hidden layers, number of hidden neurons, and I test if changing the model to a CNN would improve performance.
